env/fs_posix.cc

```diff
@@ -73,6 +76,11 @@
 #define EXT4_SUPER_MAGIC 0xEF53
 #endif
 
+#ifdef GFLAGS
+DEFINE_bool(rocksdb_io_uring_enable, true,
+            "If true, use IO uring if the platform supports it");
```

Comment:
Is this going to change the --help output of any tool using gflags that links with RocksDB? That seems intrusive